index.bs

```diff
@@ -1690,19 +1690,18 @@ To <dfn>parse attribution destinations</dfn> from a [=map=] |map|:
     [=max destinations per source=], return null.
 1. Return |result|.
 
-To <dfn>parse an expiry</dfn> given a [=map=] |map|, a [=string=] |key|, and a
-positive [=duration=] |default|:
+To <dfn>parse a duration</dfn> given a [=map=] |map|, a [=string=] |key|, a
+tuple of [=durations=] (|clampStart|, |clampEnd|):
 
-1. [=Assert=]: |default| is greater than or equal to 1 day.
-1. [=Assert=]: |default| is less than or equal to the [=max source expiry=].
 1. Let |seconds| be the result of running
     [=parse an optional 64-bit signed integer=] with |map|, |key|, and null.
```

Comment:
Should we just parse this as an unsigned integer if we're going to prohibit negative values anyway? There are slightly different considerations for the spec and the implementation (not a big deal given that the clamp range easily fits in both types), but I don't see a benefit to having the spec parse the value as signed only to error out manually.

Reply:
Would that mean a negative number returns an error rather than be clamped? That would seem breaking.

Reply:
That's exactly what this PR does anyway a few lines later.

Reply:
Oh, thanks for pointing that out! (And reminding me to read more before commenting!)